java-frontend/src/test/java/org/sonar/java/se/checks/FlowComputationTest.java

```diff
@@ -57,4 +58,10 @@ public void test_singleton() throws Exception {
     assertThat(flow).hasSize(1);
     assertThat(flow.get(0).msg).isEqualTo("singleton msg");
   }
+
+  @Ignore
+  @Test
+  public void test_multiple_paths() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)